### PR TITLE
feat(Pill): Added dragHandleComponent property to Pill component

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Added ability to override drag handle in Pill component
Added ability to set fixed width for Pill component without breaking markup.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Pill/Pill.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
   border-radius: 0.125rem;
   border: none;
   padding: 0;
+  justify-content: space-between;
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Pill/Pill.css]: @suevalov Wondering why this is needed?

Comment (m10l) [reply packages/forma-36-react-components/src/components/Pill/Pill.css]: Removing it doesn't appear to change anything in Chrome

Comment (suevalov) [reply packages/forma-36-react-components/src/components/Pill/Pill.css]: To make sure that the close button is always at the right corner. I might be wrong :)

Comment (suevalov) [reply packages/forma-36-react-components/src/components/Pill/Pill.css]: @m10l removed

**Pull request merged**